lerobot/common/policies/hilserl/classifier/configuration_classifier.py

```diff
@@ -13,7 +13,7 @@ class ClassifierConfig:
     hidden_dim: int = 256
     dropout_rate: float = 0.1
     model_name: str = "microsoft/resnet-50"
-    device: str = "cuda" if torch.cuda.is_available() else "mps"
+    device: str = "cpu"
```

Comment:
Let's use `CPU` as default, the policty should apply required device via the config